backend/src/test/java/ai/verta/modeldb/HydratedServiceTest.java

```diff
@@ -878,13 +878,10 @@ public void findHydratedExperimentRunsTest() {
             .addAllPredicates(predicates)
             // .setIdsOnly(true)
             .build();
-    try {
-      hydratedServiceBlockingStub.findHydratedExperimentRuns(findExperimentRuns);
-      fail();
-    } catch (StatusRuntimeException exc) {
-      Status status = Status.fromThrowable(exc);
-      assertEquals(Status.INVALID_ARGUMENT.getCode(), status.getCode());
-    }
+    AdvancedQueryExperimentRunsResponse hydratedResponse =
+        hydratedServiceBlockingStub.findHydratedExperimentRuns(findExperimentRuns);
+    assertEquals(
+        "Expected response not found", 0, hydratedResponse.getHydratedExperimentRunsCount());
```

Comment:
Now we are checking permission first so if given project id is not valid it will return empty list instead of PERMISSION_DENIED